WcaOnRails/.rubocop.yml

```diff
@@ -54,3 +54,9 @@ Style/SignalException:
 Style/ClassVars:
   Exclude:
     - app/models/country.rb
+
+Style/IfInsideElse:
+  Enabled: false
+
+Lint/NonLocalExitFromIterator:
+  Enabled: false
```

Comment:
Yeah, it's fairly cryptic, and there has to be a better way, but to me it's an improvement.

I'll personally put up with a lot to get away from trailing pointer iterating...